pkgs/development/python-modules/oddsprout/default.nix

```diff
@@ -30,6 +31,15 @@ buildPythonPackage rec {
     ixia
   ];
 
+  patches = [
+    # https://github.com/trag1c/oddsprout/pull/8
+    (fetchpatch2 {
+      name = "allow-periods-in-path.patch";
+      url = "https://github.com/trag1c/oddsprout/commit/09572ba2c2f045f2357c41b20646ce0420364758.patch";
```

Comment:
This should be in-tree commit https://github.com/trag1c/oddsprout/commit/59713a797e7748d48f59f92397981c93a81f2c28.patch